qiskit_experiments/data_processing/nodes.py

```diff
@@ -774,6 +774,25 @@ def _reorder(self, unordered_data: np.ndarray) -> np.ndarray:
         else:
             return unordered_data.flatten()
 
+    def _reorder_iq(self, unordered_data: np.ndarray) -> np.ndarray:
+        """Reorder IQ data according to the measurement sequence."""
+
+        if unordered_data is None:
+            return unordered_data
+
+        ordered_data = []
```

Comment:
Thanks, this makes sense, I adapted the code here: [fc55e91](https://github.com/Qiskit/qiskit-experiments/pull/752/commits/fc55e91a34cb398bf495d2002ee74d7405d4a296). I however instantiated an empty list and not an empty ndarray because populating the IQ point lists into an array did not work: 
`ValueError: setting an array element with a sequence.`